methcomp/blandaltman.py

```diff
@@ -73,27 +69,39 @@ def _check_params(self):
         if self.diff_method not in ['absolute', 'percentage']:
```

Comment:
This could be a good place to use an enum

```python
import enum, auto
...
class DIFF_METHOD(enum.Enum):
    absolute=auto()
    percentage=auto()

class BlantAltman():
    def __init__(self, diff_method: DIFF_METHOD=DIFF_METHOD.absolute, ...)
        if diff_method == DIFFMETHOD.percentage:
             ... 
```

Also possible to use with strings:
```
    if DIFF_METHOD[diff_method_string] == DIFF_METHOD.absolute:
```
will raise `KeyError` if `diff_method` not in `DIFF_METHOD`.

Reply:
While I think an enum is definitely better, I would argue that for sake of simplicity a string is good enough now.

Reply:
I think I agree